ui/packages/model3D/src/Model3DUpload.svelte

```diff
@@ -80,8 +80,8 @@
 		if (!value) return;
 
 		let url: string;
-		if (value.is_example) {
```

Comment:
Might still need `is_example`? The model3D demo does properly display the input model3d file anymore
![model_3d_upload_weird](https://user-images.githubusercontent.com/41651716/185440949-918abbb4-37a7-4faa-aadc-c94fe96df6ca.gif)

Reply:
Now fixed, thanks for catching!